recipes-core/images/intel-aero-image.bb

```diff
@@ -67,3 +67,5 @@ addtask create_link after do_rootfs before do_image
 do_create_link() {
 	ln -s lib ${WORKDIR}/rootfs/lib64
 }
+
+BAD_RECOMMENDATIONS_pn-intel-aero-image = "busybox-syslog"
```

Comment:
From manual:
"It is important to realize that if you choose to not install packages using this variable and some other packages are dependent on them (i.e. listed in a recipe's RDEPENDS variable), the OpenEmbedded build system ignores your request and will install the packages to avoid dependency errors."

Did you check it's not started/installed after this patch?